frontend/src/shared/ui/layouts/MainLayout/MainLayout.styles.ts

```diff
@@ -42,8 +42,22 @@ export const HeaderNavList = styled.ul`
 `;
 
 export const HeaderNavItem = styled.li<{ $isActive?: boolean }>`
+  display: flex;
+  align-items: center;
+  gap: 4px;
   cursor: pointer;
   color: ${(props) => (props.$isActive ? "rgba(0, 0, 0, 0.8)" : "rgba(0, 0, 0, 0.3)")};
```

Comment:
Is not critical, but you can decompose $isActive to skip props keyword, ( { $isActive } ) =>